README.md

```diff
@@ -367,8 +371,8 @@ spec:
   name: "shared-example-node-workload"
   allCores: true
   reservedCPUs:
-    - 0
-    - 1
+    - cores: [0, 1]
+      profile: "performance"
```

Comment:
Would "powerprofile" be better?

Reply:
The README if out-of-date, the field is indeed "powerprofile", will post a follow-up PR